templates/todolist-boilerplate/frontend/src/components/Tasks.tsx

```diff
@@ -28,18 +27,15 @@ export default function Tasks({
     if (!account) return;
     if (!event.target.checked) return;
     setTransactionInProgress(true);
-    const payload = {
-      type: "entry_function_payload",
-      ...createEntryPayload(ABI, {
-        function: "complete_task",
-        type_arguments: [],
-        arguments: [taskId],
-      }).rawPayload,
-    };
 
     try {
       // sign and submit transaction to chain
-      const response = await signAndSubmitTransaction(payload);
+      const response = await signAndSubmitTransaction({
```

Comment:
nite: wallet adapter soon to support sdk v2 https://github.com/aptos-labs/aptos-wallet-adapter/pull/193